.github/workflows/publish.yml

```diff
@@ -32,11 +31,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
         with:
-          fetch-depth: 0 # If you only need the current version keep this
-
-      - uses: actions/checkout@v4
-        with:
-          fetch-depth: 0 # If you only need the current version keep this
+          fetch-depth: 0
```

Comment:
Curious, does a subsequent step need access to all the tags/history?

Reply:
The only thing that needs access to the full fetch depth is actually documentation publishing. It mainly affects monorepos though. If you start like 13 releases at once, like js-core does, then it ends up that changes are being made as it builds the docs, and then has to pull and merge things.

I don't recall the original reason for the fetch-depth addition.